### PR TITLE
add a much more detailed example

### PR DESCRIPTION
**Pull request by universalmind303**
adds an example joining mongo + postgres + s3 + gcp.

Note: i didn't really know of a good directory to put this in, but think it's a pretty good example of interacting with many data sources from a real world perspective.

### CODE REVIEW

**Review by greyscaled — COMMENTED** [review]
Awesome example!!!!!!!

One question I have (question, not blocking) is if we want to try and mix 'n match `read_` in there? As in, a larger question is: where do we longer-term see the trade-off of adding data sources vs reading one-offs?

Lint & ship!

File: _glaredb/examples.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+```sql
+CREATE EXTERNAL TABLE reviews
+FROM s3
+OPTIONS (
+    region = 'us-west-2',
+    access_key_id = 'my_access_key_id',
+    secret_access_key = 'my_secret_access_key',
+    bucket = 'my_bucket',
+    location = 'reviews/*.json'
```
Comment (scsmithr) [inline _glaredb/examples.md:127]: Still needs to be added in (cc @vrongmeal )

Comment (universalmind303) [reply _glaredb/examples.md]: looks like this was merged? 
https://github.com/GlareDB/glaredb/pull/1410

Comment (vrongmeal) [reply _glaredb/examples.md]: Glob support for scan functions exists but there's an open PR for adding the same to external table. Should be in by tomorrow. (https://github.com/GlareDB/glaredb/pull/1436)

File: Gemfile.lock [hunk 2]
```diff
@@ -62,18 +60,26 @@ GEM
...
     unicode-display_width (2.4.2)
     webrick (1.8.1)
 
 PLATFORMS
   arm64-darwin-22
+<<<<<<< HEAD
+=======
   x86_64-darwin-19
   x86_64-linux
+>>>>>>> 34d41e4c978d24d1abbaca6fa82a0a94623533e5
```
Comment (greyscaled) [inline Gemfile.lock:82]: Looks like accidental commit from merge conflict

Comment (universalmind303) [reply Gemfile.lock]: sorry about that, should be good to go now!

File: Gemfile.lock [hunk 3]
```diff
@@ -62,18 +60,15 @@ GEM
...
-      rake (>= 13.0.0)
     terminal-table (3.0.2)
       unicode-display_width (>= 1.1.1, < 3)
     unicode-display_width (2.4.2)
     webrick (1.8.1)
 
 PLATFORMS
   arm64-darwin-22
-  x86_64-darwin-19
-  x86_64-linux
```
Comment (greyscaled) [inline Gemfile.lock:76]: This will break CI/deploys (see: https://github.com/GlareDB/glaredb.github.io/issues/125)

**Review by greyscaled — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 3 above)
Comment (universalmind303) [reply Gemfile.lock]: @greyscaled  I just reverted the gemfile.lock, should be good to go now.

**Review by greyscaled — APPROVED** [review]
Great write-up!

**Pull request merged**